### PR TITLE
Fix: included more languages from the language/region section

### PR DESCRIPTION
**Pull request by nyeno**
# Fixes Issue/Linear Ticket
Fix: included more languages from the language/region section

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**

# Changes proposed

## What were you told to do?

I was tasked with fixing the languages and region section. 

## What did you do?

Initially, I added only three languages. I have now updated the code to include all the transcribed languages. 

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

image

### CODE REVIEW

File: packages/main/src/components/protected/topbar/components/Profile.jsx [hunk 1]
```diff
@@ -201,6 +201,24 @@ const Profile = () => {
...
+          <div className={styles.icons}>
+            <img src={github} alt="linkedin" />
+          </div>
+          <div className={styles.icons}>
+            <img src={twitter} alt="linkedin" />
+          </div>
+          <div className={styles.icons}>
+            <img src={instagram} alt="linkedin" />
+          </div>
+            </div> */}
```
Comment (JulianaSau) [inline packages/main/src/components/protected/topbar/components/Profile.jsx:221]: Remove this commented out code if its not being used

File: packages/main/src/translations/translation-dictionary/components/Preferences.js [hunk 2]
```diff
@@ -128,14 +128,8 @@ export default [
...
     en: "Language",
-
     fr: "Langue",
-    zh: "语言",
-    ar: "لغة",
-    es: "Idioma",
-    it: "Lingua",
-    iw: "שפה",
-    pt: "Idioma"
+    de: "Taal"
```
Comment (JulianaSau) [inline packages/main/src/translations/translation-dictionary/components/Preferences.js:132]: Arent the languages you removed above being used?

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: packages/main/src/components/protected/topbar/components/Profile.jsx (hunk 1 above)
Comment (nyeno) [reply packages/main/src/components/protected/topbar/components/Profile.jsx:221]: On it right now

File: packages/main/src/translations/translation-dictionary/components/Preferences.js (hunk 2 above)
Comment (nyeno) [reply packages/main/src/translations/translation-dictionary/components/Preferences.js:132]: They are. I will add them right now

File: packages/main/src/components/protected/topbar/components/Profile.jsx (hunk 1 above)
Comment (nyeno) [reply packages/main/src/components/protected/topbar/components/Profile.jsx]: I have removed them

**Review by JulianaSau — APPROVED** [review]

**Pull request merged**